src/mantle/http/class-request.php

```diff
@@ -429,7 +435,7 @@ public function set_route_parameter( string $key, $value ) {
 	 * @return Route
 	 */
 	public function get_route(): ?Route {
-		return $this->route ?? null;
+		return isset( $this->route ) ? $this->route : null;
```

Comment:
Are we thinking this is more readable or is this to support an older PHP version or something?